### PR TITLE
Create Easel.ly.xml

### PR DESCRIPTION
**Pull request by davidabian**

### CODE REVIEW

File: src/chrome/content/rules/Easel.ly.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+		* dev.easel.ly
+		* help.easel.ly
+		* webmail.easel.ly
+
+--><ruleset name="easel.ly">
+
+	<target host="www.easel.ly" />
+
+	<rule from="^http://easel\.ly/"
+		to="https://www.easel.ly/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Easel.ly.xml]: Adding `easel.ly` as target host will make this obsolete.
^ gets secured and redirected to www, www gets then secured again.

Comment (davidabian) [inline src/chrome/content/rules/Easel.ly.xml]: Done, @J0WI. Thanks.

File: src/chrome/content/rules/Easel.ly.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<!--
+	Not supporting HTTPS or using an invalid certificate:
+		* assets.easel.ly
+		* blog.easel.ly
+		* dev.easel.ly
+		* help.easel.ly
+		* webmail.easel.ly
+
+--><ruleset name="easel.ly">
```
Comment (J0WI) [inline src/chrome/content/rules/Easel.ly.xml]: Just a minor code style issue here: Please put a line-break between `><`.

Comment (davidabian) [inline src/chrome/content/rules/Easel.ly.xml]: Sure, @J0WI. Done.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**